GoRoutines/Task-1/task.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"io"
+	"log"
+	"encoding/csv"
+)
+
+func main()  {
+	
+	column := 0
+	var colname []string
+	read := make( chan []string )
+
+	csvfile, err := os.Open("dfy.csv")	
+	if err != nil {
+		log.Fatalln("Couldn't open the csv file", err)
+	}
+	r := csv.NewReader( csvfile )
+
+	go func( column int ) {
+		for {
+			record, err := r.Read()
+
+			if err == io.EOF {
+				close(read)
+				break
+			}
+
+			if err != nil {
+				log.Fatal(err)
+				close(read)
+			}
+
+			if column == 0 {
+				colname = append(record) 
+				column++
```

Comment:
You are crossing the scope boundary. Remember, `column` is passed into the function **as value**.